### PR TITLE
New packages: rbenv-1.2.0, ruby-build-20211203

### PR DESCRIPTION
**Pull request by b-l-a-i-n-e**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-GLIBC)

### CODE REVIEW

File: srcpkgs/rbenv/INSTALL.msg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+You may verify install with rbenv-doctor
```
Comment (Chocimier) [inline srcpkgs/rbenv/INSTALL.msg:1]: This all is covered by readme. Please remove, and `vdoc` readme instead.

**Comment by b-l-a-i-n-e** [comment]
@Chocimier mind taking another look -- pushed up those changes

**Comment by Chocimier** [comment]
Make it pass lint, use xlint from xtools on template.

**Comment by b-l-a-i-n-e** [comment]
thanks @Chocimier, didn't realize there was an xtool for that-- fixed the lint errors

**Pull request merged**